aws/sdk/examples/applicationautoscaling/src/bin/describe-scaling-policies.rs

```diff
@@ -3,23 +3,43 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use applicationautoscaling::model::ServiceNamespace;
-use applicationautoscaling::{Client, Error, Region};
 use aws_config::meta::region::RegionProviderChain;
+use aws_sdk_applicationautoscaling::model::ServiceNamespace;
+use aws_sdk_applicationautoscaling::{Client, Error, Region, PKG_VERSION};
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 struct Opt {
-    /// The region
+    /// The AWS Region.
     #[structopt(short, long)]
     region: Option<String>,
 
-    /// Whether to display additional information
+    /// Whether to display additional information.
     #[structopt(short, long)]
     verbose: bool,
 }
 
-/// Lists your Amazon Cognito identities
+// Lists the Application Auto Scaling policies.
+async fn show_policies(
+    client: &aws_sdk_applicationautoscaling::Client,
+) -> Result<(), aws_sdk_applicationautoscaling::Error> {
```

Comment:
This error type is already imported:
```suggestion
) -> Result<(), Error> {
```

Reply:
Good catch, fixed.